src/briefcase/macos.py

```diff
@@ -58,6 +62,14 @@ def launcher_script_location(self):
         return macos_dir
 
     def build_app(self):
+        print("Building DMG file...")
+        dmg_name = self.formal_name + '.dmg'
+        dmg_path = os.path.join(os.path.abspath(self.dir), dmg_name)
+        settings = {'files': [self.app_location],
+                    'symlinks': {'Applications': '/Applications'}}
+        dmgbuild.build_dmg(filename=dmg_path,
+                           volume_name=self.formal_name, settings=settings)
```

Comment:
At column 87: (E262) inline comment should start with '# '

Reply:
At least it's consistent with itself but I don't see any thing that looks like a comment here